cypress/e2e/pages/release.cy.ts

```diff
@@ -17,7 +18,7 @@ describe('Release Page', () => {
     before(() => {
       cy.findByRole('button', { name: /Release Labware/i })
         .should('be.visible')
-        .click();
+        .click({ force: true });
```

Comment:
The tests are failing in CI and the flakiness in CI is usually of slow network or slow DOM updates when compared to running it locally mainly of timing issues. This is one of the way to ensure that it will resolve it by waiting for button to be visible. Though this is suggested in many places, it is not a perfect solution. It is better to use good patterns by relying on built-in retry ability which allows to wait for the application to finish updating which is enforced by checking for e.g should('be.visible').